lib/engine/connection.rb

```diff
@@ -5,39 +5,25 @@ class Connection
     attr_reader :paths
 
     def self.connect!(hex)
-      connections = {}
-      node_paths = []
-      hex_edges = {}
-
       hex.tile.paths.each do |path|
-        path.walk { |p| node_paths << p if p.node? }
-      end
-
-      node_paths.uniq.each do |node_path|
-        node_path.walk(chain: []) do |chain|
+        path.walk(chain: []) do |chain|
```

Comment:
why aren't we walking node paths anymore? why do we walk all paths?

Reply:
That was really for when an tile without nodes was laid. It'd then update all the hexes connected. The walk paths still happens but only to invalidate the cache